templates/zerver/reset_confirm.html

```diff
@@ -63,7 +65,7 @@ <h1>{{ _('Set a new password.') }}</h1>
 
                 <div class="input-box m-t-30">
                     <div class="centered-button">
-                        <button type="submit" class="" value="Submit">Submit</button>
+                        <button type="submit" class="register-button" value="Submit">Submit</button>
```

Comment:
`submit-button` would be a better class here, so you can add the same class in `register.html` as well. Also, would good to use form IDs in CSS as well to avoid breaking any other UI as `submit-button` is somehwat generic class name.